xc7/primitives/slicem/Ndram/dpram64.pb_type.xml

```diff
@@ -19,24 +19,24 @@
   <input  name="WA"  num_pins="6" />
   <input  name="WA7" num_pins="1" />
   <input  name="WA8" num_pins="1" />
-  <input  name="DI1" num_pins="1" />
+  <input  name="DI" num_pins="1" />
```

Comment:
Why change this?  The name is correct?

Reply:
There's only one DI in a DPRAM, which usually maps to DI1, except it can be DI2 in 32 bit mode.